media-api/app/lib/elasticsearch/ElasticSearch.scala

```diff
@@ -91,14 +91,39 @@ object ElasticSearch extends ElasticSearchClient with SearchFilters with ImageFi
     val missingIdentifier= params.missingIdentifier.map(idName => filters.missing(NonEmptyList(identifierField(idName))))
     val uploadedByFilter = params.uploadedBy.map(uploadedBy => filters.terms("uploadedBy", NonEmptyList(uploadedBy)))
 
-    val validityFilter   = params.valid.flatMap(valid => if(valid) validFilter else invalidFilter)
+    val validityFilter: Option[FilterBuilder] = params.valid.flatMap(valid => if(valid) validFilter else invalidFilter)
 
     val costFilter       =
       if (params.costModelDiff) params.free.flatMap(free => if (free) freeDiffFilter else nonFreeDiffFilter)
       else                      params.free.flatMap(free => if (free) freeFilter else nonFreeFilter)
 
+
+    val persistedFilter = params.persisted map {
```

Comment:
Just to keep this file more readable / expressive of what we're doing, leaving the actual implementation in the filters file. e.g.
```
case true => true persistedFilter
case false => notPersistedFilter
```
Makes easily readable sense.